sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -31,7 +31,7 @@ class WordCountIT(unittest.TestCase):
   @attr('IT')
   def test_wordcount_it(self):
     # Set extra options to the pipeline for test purpose
-    extra_opts = {'on_success_matcher': PipelineStateMatcher()}
```

Comment:
right, Pyhamcrest `any_of` and `all_of` do exactly what I want in this PR, but in a cleaner way. Thanks for suggestion!